grype/vulnerability/vulnerability.go

```diff
@@ -13,6 +13,7 @@ type Vulnerability struct {
 	CPEs           []pkg.CPE
 	ID             string
 	RecordSource   string
+	Namespace      string
```

Comment:
Since I saw `Namespace` here, I assumed the rest would follow with the same convention. My preference would be to use `Namespace` throughout, but if we are not, then normalize for `grypeDbNamespace`. Keeping both reduces clarity

Reply:
renamed back to `namespace` 👍